### DISCUSSION TITLE
Can I make this nav bar (not sure that;'s the right term) smaller, say 30px

### DISCUSSION THREAD

**Discussion by pitosalas** [squidfunk/mkdocs-material#6659] [Q&A] [answered]
I am not sure it's called the nav bar, but here is what I mean:

<img width="1235" alt="Screenshot 2024-01-17 at 12 16 47 PM" src="https://github.com/squidfunk/mkdocs-material/assets/44681/6f6abe0e-7b4b-4fc3-b046-e3a7f924b6c1">

I would like to make it thinner, say around 30px. I tried various css things but none of them worked fully. And I can't find an official parameter for that. Any idea?

**Comment by alexvoss** [comment, marked as answer]
There is not setting for most of these things, for the simple reason that making everything configurable would lead to bloat - and then there would still be thing people want to have configurable. Instead, the approach is that you can configure the elements of the site by editing templates, partial and blocks. The process is described in [the customization guide](https://squidfunk.github.io/mkdocs-material/customization/#customization). 

You can then add your own CSS styling on top of this with [extra css](https://squidfunk.github.io/mkdocs-material/customization/#additional-css). Your browser's developer tools are your friend here for figuring out what classes you can attach to (this is assuming that you are not familiar with SCSS and can interpret the source directly).

In this case, the main problem is going to be re-arranging or resizing the content of the header. Just setting its height to 30px will not look good. If you are interested in a more lightweight look rather than saving space, can I suggest that you change the colors used instead of changing the layout? I think that should be comparatively easy to do. You can also use an image as a background, as [described in this discussion](https://github.com/squidfunk/mkdocs-material/discussions/6654).

Hope this helps.

**Comment by pitosalas** [comment]
Thanks, I did change the color and it helped just like what you said!
